vast/defaults/defaults.go

```diff
@@ -2,13 +2,23 @@ package defaults
 
 import "time"
 
-const MAX_VIDEO_DURATION = 60 * time.Second
-const MIN_VIDEO_DURATION = 1 * time.Second
+// MaxVideoDuration stand for the max video duration with a default of 60 seconds
```

Comment:
stand->stands, same for others.

Reply:
Replace 'stand' with 'stands' to fix this grammar error.